.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Publish
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        node-version: [lts/*]
```

Comment:
Can you also add the older Node.js versions that `sox-static` currently supports, i.e. >= 6?